src/it/scala/uk/gov/hmrc/support/TemplateIntegrationBaseSpec.scala

```diff
@@ -83,6 +84,8 @@ abstract class TemplateIntegrationBaseSpec[T: OWrites: Arbitrary](
             println("Skipping property evaluation")
 
             true
+          case Failure(exception)                            =>
+            throw exception
```

Comment:
added this to suppress exhaustiveness warning but still throw on unexpected errors